addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -368,6 +372,59 @@ const Wysiwyg = Widget.extend({
     redo: function () {
         this.odooEditor.historyRedo();
     },
+    /**
+     * Start or resume the date field changes muation observers.
+     *
+     * Necessary to keep all copies of a given field at the same value throughout the page.
+     */
+    _observeDateFieldChanges: function () {
+        if(this.dateObservers) {
+            for (let observerData of this.dateObservers) {
+                observerData.observer.observe(
+                    observerData.field,
+                    { characterData:true, subtree: true, childList: true }
```

Comment:
I would place the params in a constant as it is used two times.

Reply:
:+1